src/style.css

```diff
@@ -1 +1,25 @@
 /* add styles here */
+body {
+  margin: 0;
```

Comment:
You are using the body tag for styling. It's recommended to use classes instead of tag names for styling (except for html and body). So, this is acceptable in this case.